Source/hydro/riemann.H

```diff
@@ -54,7 +54,7 @@ Castro::pstar_bisection(Real& pstar_lo, Real& pstar_hi,
                         const Real gdot, const Real gmin, const Real gmax,
                         const int lcg_maxiter, const Real lcg_tol,
                         Real& pstar, Real& gamstar,
-                        bool& converged, GpuArray<Real, PSTAR_BISECT_FACTOR*HISTORY_SIZE>& pstar_hist_extra) {
+                        bool& converged, GpuArray<Real, PSTAR_BISECT_FACTOR*HISTORY_SIZE> pstar_hist_extra) {
```

Comment:
Don't we want to keep this by reference since we're writing to `pstar_hist_extra`?

Reply:
changed